src/main/scala/mesosphere/marathon/raml/VolumeConversion.scala

```diff
@@ -8,13 +8,20 @@ import org.apache.mesos.{Protos => Mesos}
 
 trait VolumeConversion extends ConstraintConversion with DefaultConversions {
 
+  /**
+    * If a disk profile is set, disk type defaults to Mount, as only that is supported by DSS which sets profileNames
+    * In any other case, DiskType defaults to Root
```

Comment:
There is so much context missing... what is DSS?  why does it use Mount?   What is the difference between Mount and Root.    IMO I think it is better to stay away from DCOS specific language.   I would express that there are X services which use Mount and Y services that use ROOT (and why or benefit).   I would also express that the detection of Mount is that it uses `profileNames` and Root does not.

Reply:
Again, very good point, @kensipe. I'll be much more verbose here, and provide links for further reading.